### PR TITLE
feat: config library, storybook

### PR DESCRIPTION
**Pull request by ngochuyduong**
To get started with the project, run `yarn` in the root directory to install the required dependencies for each package:
yarn

While developing, you can run the [example app](/example/) to test your changes. Any changes you make in your library's JavaScript code will be reflected in the example app without a rebuild. If you change any native code, then you'll need to rebuild the example app.

To start the packager:
yarn example start

To run the example app on Android:
yarn example android

To run the example app on iOS:
yarn example ios

Screenshot

Screenshot2

### CODE REVIEW

**Comment by baonguyenhsts** [comment]
I am questioning whether we require ios and android folders in addition to the example folder for this base component. Since the component is intended for React Native. I inquired with @ngochuyduong, and he confirmed that the ios and android folders contain the View component.
@honghoangsts @ngochuyduong

**Comment by ngochuyduong** [comment]
> I am questioning whether we require ios and android folders in addition to the example folder for this base component. Since the component is intended for React Native. I inquired with @ngochuyduong, and he confirmed that the ios and android folders contain the View component. @honghoangsts @ngochuyduong

Yes. The iOS, Android folder in the root folder is for native view component.

File: rn-base-component.podspec [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Pod::Spec.new do |s|
+  s.name         = "rn-base-component"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => "11.0" }
+  s.source       = { :git => "https://github.com/ngochuyduong/rn-base-component.git", :tag => "#{s.version}" }
```
Comment (loc-nguyenthien) [inline rn-base-component.podspec:15]: Change the git link to our main github => `https://github.com/saigon-technology/rn-base-component.git`

File: src/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  text: string
+  color?: string
+  textColor?: string
+}
+
+const styles = StyleSheet.create({
+  button: {
+    paddingVertical: 8,
+    paddingHorizontal: 16,
+    borderRadius: 4,
```
Comment (loc-nguyenthien) [inline src/Button.tsx:15]: Please use `metrics` values

File: example/src/stories/Button.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+// stories/MyButton.stories.tsx
+import React from 'react'
+import type {ComponentMeta, ComponentStory} from '@storybook/react'
+
+import {RnBaseButton} from 'rn-base-component'
+
+export default {
+  title: 'components/MyButton',
```
Comment (loc-nguyenthien) [inline example/src/stories/Button.stories.tsx:8]: I think we should change it to `components/Button`

File: src/Button.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    <TouchableOpacity
+      style={[styles.button, !!color && {backgroundColor: color}]}
+      onPress={onPress}
+      activeOpacity={0.8}>
+      <Text style={[styles.buttonText, !!textColor && {color: textColor}]}>{text}</Text>
+    </TouchableOpacity>
+  </View>
+)
+
+export default MyButton
```
Comment (loc-nguyenthien) [inline src/Button.tsx:42]: I think we should change it to `Button`

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]

File: rn-base-component.podspec (hunk 1 above)
Comment (ngochuyduong) [reply rn-base-component.podspec]: updated

**Review by chungnguyenhoang — APPROVED** [review]

File: rn-base-component.podspec (hunk 1 above)
Comment (loc-nguyenthien) [reply rn-base-component.podspec]: Done

**Review by loc-nguyenthien — APPROVED** [review]

**Pull request merged**